packages/website/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@web3-storage/website",
+  "version": "0.0.0",
+  "description": "web3.storage website",
+  "private": true,
+  "scripts": {
+    "start": "next dev",
+    "build": "next build && next export",
+    "test": "eslint './**/*.js' && tsc --build"
+  },
+  "dependencies": {
+    "@magic-ext/oauth": "^0.7.0",
+    "clsx": "^1.1.1",
+    "filesize": "^6.1.0",
+    "magic-sdk": "^4.2.1",
+    "next": "^11.0.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-if": "^4.0.1",
+    "react-query": "^3.13.9",
+    "tachyons": "^4.12.0",
+    "web3.storage": "^0.0.0"
+  },
+  "devDependencies": {
+    "@types/react": "^17.0.4",
+    "eslint": "^7.24.0",
+    "eslint-config-prettier": "^8.2.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-react": "^7.23.2",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "typescript": "^4.3.5"
+  },
+  "eslintConfig": {
+    "extends": [
+      "standard",
+      "plugin:react/recommended",
+      "plugin:react-hooks/recommended",
+      "prettier"
+    ],
+    "env": {
+      "es2020": true,
+      "browser": true,
+      "node": true
+    },
+    "rules": {
+      "react/prop-types": "off",
+      "no-unused-vars": [
+        "error",
+        {
+          "args": "all",
+          "argsIgnorePattern": "_",
+          "varsIgnorePattern": "_"
+        }
+      ],
+      "react/react-in-jsx-scope": "off",
+      "react/jsx-filename-extension": [
+        1,
+        {
+          "extensions": [
+            ".js",
+            ".jsx"
+          ]
+        }
+      ],
+      "react/display-name": 1
+    },
+    "settings": {
+      "react": {
+        "version": "detect"
+      }
+    }
+  },
```

Comment:
note to self: we should unify this with the prettier config at the root of the project